mon-pix/app/templates/components/course-banner.hbs

```diff
@@ -3,7 +3,12 @@
   <h1 class="course-banner__name">{{courseName}}</h1>
 
   {{#if withHomeLink}}
-    {{#link-to "index" class="course-banner__home-link"}}Revenir à l'accueil{{/link-to}}
+    {{#if course.isDemo}}
```

Comment:
La logique est sensiblement la même que dans result.hbs du dessus ... à quel point pourrait-on mutualiser ce code ? Possibilité d'en faire un composant à qui on passe le style adapté à la page dans laquelle il se trouve ?

Reply:
Parfois c'est un style différent, d'autres fois, un link-to, ou un <a>. J'ai l'impression que si je mutualise, ca fera un composant assez compliqué à comprendre..